doc/2/framework/classes/request/setResult/index.md

```diff
@@ -0,0 +1,54 @@
+---
+code: true
+type: page
+title: setResult
+description: Request class setResult() method
+---
+
+# setResult
+
+Sets the request result and status
+
+### Arguments
+
+```ts
+setResult(
+  result: any,
+  options: {
+    status?: number
+    headers?: JSONObject;
+    raw?: boolean;
+  }): void;
+```
+
+</br>
+
+| Name | Type | Description                      |
+|------|------|----------------------------------|
+| `result` | <pre>any</pre> | Request result. Will be converted to JSON unless `raw` option is set to `true` |
+| `options` | <pre>JSONObject</pre> | Optional parameters |
+
+The `options` argument may contain the following properties:
+
+| Name | Type | Description                      | Default |
+|------|------|----------------------------------|---------|
+| `status` | <pre>integer</pre> | HTTP status code | `200` |
+| `headers` | <pre>JSONObject</pre> | Additional response protocol headers | `null` |
+| `raw` | <pre>boolean</pre> | Instead of a Kuzzle response, forward the result directly | `false` |
```

Comment:
```suggestion
| `raw` | <pre>boolean</pre> | Instead of a Kuzzle response, forward the result directly to the client, without being converted to an API response payload (can be used to answer in a different format than JSON) | `false` |
```